plugins/inputs/mqtt_consumer/README.md

```diff
@@ -23,6 +23,20 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 [CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
 
+## Startup error behavior options <!-- @/docs/includes/startup_error_behavior.md -->
+
+In addition to the plugin-specific and global configuration settings the plugin
+supports options for specifying the behavior when experiencing startup errors
+using the `startup_error_behavior` setting. Available values are:
+
+- `error`:  Telegraf with stop and exit in case of startup errors. This is the
+            default behavior.
+- `ignore`: Telegraf will ignore startup errors for this plugin and disables it
+            but continues processing for all other plugins.
+- `retry`:  Telegraf will try to startup the plugin in every gather or write
+            cycle in case of startup errors. The plugin is disabled until
```

Comment:
I think there is something missing in this sentence, was it meant like that?
```suggestion
- `retry`:  Telegraf will try to startup the plugin in every cycle and gather or write
            in case of startup errors. The plugin is disabled until
```

Reply:
I think the original statement is correct and your suggestion has a different meaning... For inputs, we try to reconnect in every `Gather()` call (i.e. gather cycle) and for output plugins in every `Write()` call (i.e. write cycle). :-)